packages/client/src/pages/ForumPage/index.tsx

```diff
@@ -0,0 +1,304 @@
+              },
+              "& .MuiOutlinedInput-notchedOutline": {
+                borderColor: "gray",
+              },
+            }}
+          />
+          <Box sx={{ mt: 3 }}>
+            <Box sx={{ display: "flex", gap: 1, mb: 1 }}>
+              <IconButton
+                onClick={() => setFormatting(prev => ({ ...prev, bold: !prev.bold }))}
+                color={formatting.bold ? "primary" : "default"}>
+                <FormatBoldIcon />
+              </IconButton>
+              <IconButton
+                onClick={() => setFormatting(prev => ({ ...prev, italic: !prev.italic }))}
+                color={formatting.italic ? "primary" : "default"}>
+                <FormatItalicIcon />
+              </IconButton>
+              <IconButton
+                onClick={() => setFormatting(prev => ({ ...prev, underline: !prev.underline }))}
+                color={formatting.underline ? "primary" : "default"}>
+                <FormatUnderlinedIcon />
+              </IconButton>
+            </Box>
+            <TextField
+              fullWidth
+              placeholder="Описание"
+              multiline
+              rows={4}
+              variant="outlined"
+              value={description}
+              onChange={e => setDescription(e.target.value)}
+              sx={{
+                "mb": 2,
+                "& .MuiOutlinedInput-root": {
+                  borderRadius: "12px",
+                  borderColor: "gray",
+                  backgroundColor: "transparent",
+                },
+                "& .MuiOutlinedInput-notchedOutline": {
+                  borderColor: "gray",
+                },
+              }}
+            />
+          </Box>
+          <UiButton fullWidth onClick={handleCreate}>
+            Создать тему
+          </UiButton>
+        </UiPaper>
+      </Box>
+    </Modal>
+  );
+};
+
+export function ForumPage() {
+  const navigate = useNavigate();
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [topics, setTopics] = useState<Topic[]>([
+    {
+      id: 1,
+      title: "Обсуждение стратегии",
+      author: "Игрок1",
+      messages: 15,
+      date: "2025-01-20",
+      description: "",
+    },
+    {
+      id: 2,
+      title: "Ошибки и баги",
+      author: "Игрок2",
+      messages: 8,
+      date: "2025-01-18",
+      description: "",
+    },
+    {
+      id: 3,
+      title: "Турниры и соревнования",
+      author: "Игрок3",
+      messages: 12,
+      date: "2025-01-19",
+      description: "",
+    },
+    {
+      id: 4,
+      title: "Лучшие результаты",
+      author: "Игрок4",
+      messages: 20,
+      date: "2025-01-17",
+      description: "",
+    },
+  ]);
+
+  const handleOpenModal = () => setIsModalOpen(true);
+  const handleCloseModal = () => setIsModalOpen(false);
+
+  const handleCreateTopic = (newTopic: Omit<Topic, "id" | "messages" | "date">) => {
+    const date = new Date().toISOString().split("T")[0];
```

Comment:
можно лучше:
```suggestion
    const date = new Date().toLocaleDateString("ru-RU");
```

так намного проще и меньше вероятность получить баг